docs/content/quickstart/parameters.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "QuickStart: Parameters"
+descriptions: Learn how to use a bundle with parameters
+layout: single
+---
+
+Now that you know how to install a bundle, let's look at how to specify parameters to customize how that bundle is installed.
+Bundles can define parameters to allow the end-user to tweak how the bundle is configured.
+A parameter can be a string, integer, boolean or even a json object.
+Some examples of how parameters can be used in a bundle are:
+
+* Log Level: Default the log level for an application to info. At any time a user can upgrade the bundle to change that parameter to a different value.
+* Deployment Region: Let the user specify which region, such as eastus-1, where the application should be deployed.
+* Helm Release Name: A bundle that uses Helm often will define a parameter that allows the user to set the release name for the Helm release.
+
+For optional parameters, bundles set a default value that is used when the user does not specify a value for the parameter.
+
+Let's look at a bundle with parameters:
+
```

Comment:
I'm wondering if starting this out looking at the same bundle that was used in the quickstart (getporter/wordpress:v0.1..3) would be useful here.

Reply:
I decided to not use that bundle because it requires credentials, which we have not introduced yet, and a kubernetes cluster (more work for the reader that isn't relevant).

I used wordpress as an example earlier in the first quickstart because it is a bundle that has credentials, parameters, outputs and custom actions defined. That was useful because I was highlighting the output of porter explain, and if all the output said was "no credentials/parameters/outputs defined" then that wouldn't demonstrate the command very well. This is why later in the same quickstart I switched to the hello world bundle, so that we weren't using features that have not been explained yet.